client/state/rewind/size/reducer.ts

```diff
@@ -65,15 +66,15 @@ const daysOfBackupsSaved = (
 	return size.daysOfBackupsSaved ?? null;
 };
 
-const retentionDays = (
-	state: AppState = null,
-	{ type, size }: AnyAction
-): AppState | number | null => {
-	if ( type !== REWIND_SIZE_SET ) {
-		return state;
+const retentionDays = ( state: AppState = null, payload: AnyAction ): AppState | number | null => {
```

Comment:
I'd rename `payload` to `action` since it actually is an action and the `payload` is usually only part of the action.

Reply:
Makes sense! Just changed it :D